docs/odata/v2vsv4.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: V2 vs. V4
+name: vs2vsv4
+parent: oData
+nav_order: 2
+---
+
+## Major Differences
+
+These are the major differences between oData V4 and oData V4:
+
+* In OData 4, the JSON data format now allows to control the amount of metadata that is returned in query responses. There are three levels of metadata supported [(see details)](http://docs.oasis-open.org/odata/odata-json-format/v4.0/os/odata-json-format-v4.0-os.html#_Toc372793040):
+  * _Full_: The response contains all the metadata needed to describe the response.
+  * _Minimal_: The response metadata references the metadata document. Information in the metadata document is not repeated in the response.
+  * _None_: The response contains no metadata. The application must understand the response structure.
+
+* OData 4 adds a new flexible search capability, `$search`. The search feature allows you to query a collection for entities that match a specified search expression. Unlike the existing filter capability, which allows a query to specify that a specific property or properties match certain criteria, the search feature can apply the search expression to any of the properties of an entity.
+
+* The `$expand` system query option has been enhanced in OData 4. This feature specifies the related resources to be included in line with retrieved resources. In OData 2, if a single value navigation property is expanded, you get all the properties of the entity if it was a single value navigation property. And if a collection navigation property is expanded, you get all of the entities in the collection and all of the properties of those entities. In OData 4, you can now refine the results using the `$select`, `*`, `$filter` and `$top` operations.
+
+* $count replaces `$inlinecount` in OData 4. `$count` has been enhanced to be used with `$filter`, `$expand` and `$orderby` options.
+
+* Changes in support for data types in OData 4:
+  * `Edm.DateTime` has been deprecated. The lack of timezone information in OData 2 causes significant problems. Use `Edm.DateTimeOffset` instead.
+  * `Edm.Time` has been replaced with `Edm.Duration` and `Edm.TimeOfDay` to make it clear whether it is duration of a specific time of day.
+  * `Edm.Date` has been added as there was no way to express just a date in OData 2.
+  * `Edm.Float` has been eliminated.
+
+* The oData V4 model does not support the methods `getData`, `getObject`, `getOriginalProperty`, `getProperty`. For data access, use the context API instead of methods on the model.
```

Comment:
"OData" :)
plus it's the _UI5_ OData v4 model (might help to add the full namespace sap.ui.model....), not OData v4 per se